docs/apps/matlab.md

```diff
@@ -0,0 +1,197 @@
+# MATLAB
+
+MATLAB is a high-level technical computing language and interactive environment for algorithm development, data visualization, data analysis, and numeric computation.
+
+- High-level language for numerical computation, visualization, and application development.
+- Interactive environment for iterative exploration, design, and problem solving.
+- Mathematical functions for linear algebra, statistics, Fourier analysis, filtering, optimization, numerical integration, and solving ordinary differential equations.
+- Built-in graphics for visualizing data and tools for creating custom plots.
+- Development tools for improving code quality and maintainability and maximizing performance.
+- Tools for building applications with custom graphical interfaces.
+- Functions for integrating MATLAB based algorithms with external applications and languages such as C and Java.
+
+[TOC]
+
+## Available
+
+- Puhti: R2019b, R2019a, R2018b, R2018a, R2017b
+
+## License
+
+Proprietary software. The terms of use of this software allow its use for only the affiliates (staff and students) of Finnish higher education institutions. **NB** If you are an user from a commercial company or Finnish research institute, please contact CSC servicedesk for further instructions.
+
+## Usage
+
+At CSC, MATLAB is available both interactive and batch jobs. The interactive sessions are intended for light pre- and postprocessing of data, whereas larger parallel jobs should be run via batch job system of Puhti using MATLAB Parallel Server (MPS) tool.
+
+### Interactive MATLAB Sessions on Puhti
+<div id="interactive-matlab" />
+
+There are four interactive MATLAB licenses with two Parallel Computing Toolbox and one Compiler license available for temporary interactive academic use. We recommend to use [NoMachine](nomachine.md) client for the sessions. After logging in to Puhti, MATLAB can be launched as follows:
+
+```bash
+$ module load matlab/R2018b
```

Comment:
Module is not loadable
'ls -ld /appl/modulefiles/matlab/'
**drwx--S---. 2 ssillanp p_installation 4096 Aug 12 09:39**

Reply:
Fixed. Thanks!